moveit_ros/planning/constraint_sampler_manager_loader/CMakeLists.txt

```diff
@@ -2,7 +2,14 @@ set(MOVEIT_LIB_NAME moveit_constraint_sampler_manager_loader)
 
 add_library(${MOVEIT_LIB_NAME} src/constraint_sampler_manager_loader.cpp)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES})
+target_link_libraries(${MOVEIT_LIB_NAME} 
```

Comment:
`ament_target_dependencies()` and add Boost, pluginlib, moveit_core (because of moveit_macros).